Shared/Features/Freaks/View/FreaksButtonsView.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+
+struct FreaksButtonsView: View {
```

Comment:
I think a better name would be FilterButtonsView. In this way, we have more context about what these buttons are for.